`#[derive(ToSchema)]` fails on `time::Date` fields

Repro:

```rust
use time; // 0.3.17
use utoipa; // 2.4.2, features = ["time"]

#[derive(utoipa::ToSchema)]
struct Foo {
    bar: time::Date,
}
```

Expanded:

```rust
struct Foo {
    bar: time::Date,
}
impl utoipa::ToSchema for Foo {
    fn schema() -> utoipa::openapi::schema::Schema {
        utoipa::openapi::ObjectBuilder::new()
            .property(
                "bar",
                utoipa::openapi::ObjectBuilder::new()
                    .schema_type(utoipa::openapi::SchemaType::String)
                    .format(Some()),                                          // <- note the "Some()"
            )
            .required("bar")
            .into()
    }
}
```

Error:

```
error[E0061]: this enum variant takes 1 argument but 0 arguments were supplied
   --> src/lib.rs:12:29
    |
12  |                     .format(Some()),
    |                             ^^^^-- an argument of type `SchemaFormat` is missing
    |
note: tuple variant defined here
   --> /Users/kinrany/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/core/src/option.rs:526:5
    |
526 |     Some(#[stable(feature = "rust1", since = "1.0.0")] T),
    |     ^^^^
help: provide the argument
    |
12  |                     .format(Some(/* SchemaFormat */)),
    |                                 ~~~~~~~~~~~~~~~~~~~~

For more information about this error, try `rustc --explain E0061`.
error: could not compile `repro-utoipa-time-toschema-derive` due to previous error
```

~~As far as I understand, the correct macro output in this case would be `Some(SchemaFormat::KnownFormat(KnownFormat::Date))`.~~

--- a/utoipa-gen/src/schema_type.rs
+++ b/utoipa-gen/src/schema_type.rs
@@ -285,7 +285,7 @@ impl ToTokens for Type<'_> {
             "f32" | "f64" => tokens.extend(quote! { utoipa::openapi::SchemaFormat::KnownFormat(utoipa::openapi::KnownFormat::Float) }),
             #[cfg(feature = "chrono")]
             "DateTime" => tokens.extend(quote! { utoipa::openapi::SchemaFormat::KnownFormat(utoipa::openapi::KnownFormat::DateTime) }),
-            #[cfg(any(feature = "chrono", feature = "Time"))]
+            #[cfg(any(feature = "chrono", feature = "time"))]
             "Date" => tokens.extend(quote! { utoipa::openapi::SchemaFormat::KnownFormat(utoipa::openapi::KnownFormat::Date) }),
             #[cfg(feature = "uuid")]
             "Uuid" => tokens.extend(quote! { utoipa::openapi::SchemaFormat::KnownFormat(utoipa::openapi::KnownFormat::Uuid) }),

